keanu-project/src/main/java/io/improbable/keanu/vertices/LogProbGraphSupplier.java

```diff
@@ -0,0 +1,7 @@
+package io.improbable.keanu.vertices;
+
+public interface LogProbGraphSupplier {
```

Comment:
Is this interface just temporary? Once we've implemented logProb graphs throughout, I'd expect this to be part of the `Probabilistic` interface

Reply:
Yeah it's just temporary